### ISSUE TITLE
Domänenwissen aneignen

### ISSUE THREAD

**Issue by yvokeller** [Sensor-Based-Activity-Recognition/docs#3]
Papers zu Activity Recognition

- [x] Template um wichtige Paper Infos & Resultate festzuhalten
- [x] Papers zusammenfassen (inkl. Feature Extraction)
- [x] F1 Macro Score -> Loss Function auf F1-Marco Metrik möglich? -> "The macro F1-score has one big trouble. It's non-differentiable. Which means we cannot use it as a loss function."

**Festhalten**
- Methodik
- Hyperparameter
- Interessante Modelle
- Was wurde erreicht

**Comment by gabrieltorresgamez** [comment]
Bitte auch mögliche Feature Extraction festhalten

**Comment by yvokeller** [comment]
@buesst1 nicht vergessen auch ein paper anschauen :D
